src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -447,12 +496,22 @@
     /** Which path comparator to use */
     public String pathComparator = null;
 
+    public boolean enterStationsWithCar = false;
```

Comment:
Please add Javadoc about why this is needed - at first glance it appears that it would just be the negation of a boolean indicating whether we're using park and ride, but I get the impression there's some reason "not using park and ride" is not sufficient information.

Reply:
In flex routing, a few searches before the main search interrogate the graph and then temporary edges/vertices are created. The prelininary search for call-and-ride happens in CAR mode, since it models a vehicle traversing the road network. In order to ultimately get itineraries where call-and-ride is taken to transfer to fixed-route, the preliminary search needs to be able to discover TransitStops. This variable allows that to happen.

Some alternatives to this new parameter are:
-  Using the kiss-and-ride parameter, as you imply in a different comment. The preliminary searches would need to start in the "car parked" state for this to work. This seems a bit hacky (ie it's basically abusing the parameters, we're not really doing a kiss-and-ride search with a parked car) and it would require a new State constructor to set carParked=true in the initial State.
- The preliminary searches could interrogate the StreetTransitLink edge that's discovered, rather than the TransitStop vertex. This would be a bit awkward, but doable.

I'll add documentation to this parameter - but is there a compelling reason to use either of the approaches above instead?